### DISCUSSION TITLE
Cannot build sites: error rendering 404/calling markdownify

### DISCUSSION THREAD

**Discussion by buson94** [nunocoracao/blowfish#704] [Q&A]
**Describe the bug**
I fear this is most likely a problem on my side, with the version of hugo i use or how i set up my project but just in case it isn't: If i try to build my sites using `hugo` in my visual studio code terminal (powershell) i get a runtime error `invalid memory address or nil pointer dereference` when trying to render "404".

**To Reproduce**
1. Created GitLab Repo using the *Pages/Hugo* Template
2. Pulled the Repo through Visual Studio Codes Git Extension
3. Added blowfish as a git submodule
4. Deleted default config.toml
5. Added config files from blowfish folder

**Expected behavior**
The sites being able to be build.

**Screenshots**
![blowfish_error](https://user-images.githubusercontent.com/11974889/236165764-cfecfbfd-5ded-4579-89c9-8c4b4d0f88ef.jpg)

**Desktop (please complete the following information):**
 - OS: Windows 10 Version 22H2 Build 19045.2788
 - IDE: VS Code 1.78.0

**Hugo & Blowfish versions**
 - Hugo extended v0.111.3
 - Blowfish v2.33.0

**Comment by nunocoracao** [comment]
Hey @buson94 moving this into a discussion. Here are my thoughts (I don't have an equivalent OS to test this - I am on Mac) 
this `Pages/Hugo Template` seems to be the problem. I would try a local test using the Hugo install guide and then Blowfish. If that works the template is at fault. Might be wrong though. In my experience, I've run into several problems everytime I've tried to use templates for Hugo.

**Reply by buson94** [reply]
So... it was fully unrelated to OS or template being used but instead the root _index.md having `draft: true` in the front matter. This variable was still existent from when i followed the [official tutorial](https://gohugo.io/getting-started/quick-start/#add-content) and it not mentioning the variable should be `false`/deleted/non-existent in the first place on the roots index page before building. 

Someone may say this should be a no-brainer but since I'm new to static site generators and am still experimenting with it, i didn't think of that until now 😅

Nonetheless very weird error message for this case, can't wrap my head around why it can't render the 404 page then 🤔
